cypress/e2e/specialFeatures/cookieBanner/index.cy.js

```diff
@@ -1,54 +1,169 @@
+    service: 'mundo',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/mundo',
+    service: 'mundo',
+    runforEnv: ['local', 'test', 'live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/mundo/popular/read',
+    service: 'mundo',
+    runforEnv: ['local', 'test', 'live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/mundo/media-52123665',
+    service: 'mundo',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/mundo/articles/ce42wzqr2mko',
+    service: 'mundo',
+    runforEnv: ['local', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/mundo/media-23283126',
+    service: 'mundo',
+    runforEnv: ['local', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/liveradio', // Live Radio,
+    service: 'pashto',
+    runforEnv: ['local', 'test', 'live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/multimedia/2016/08/160827_gn_southafrica_female_farmer', // TC2 MAP
+    service: 'pashto',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/world-52873295', // CPS STY
+    service: 'pashto',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/arts-and-literature-50230813', // PGL
+    service: 'pashto',
+    runforEnv: ['live', 'local'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/programmes/p0340yr4', // On Demand Radio Brand
+    service: 'pashto',
+    runforEnv: ['live', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/world/2016/09/160921_tc2_testmap1?renderer_env=test', // TC2 MAP
+    service: 'pashto',
+    runforEnv: ['local', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/23289748', // CPS STY
+    service: 'pashto',
+    runforEnv: ['local', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/23092924', // CPS PGL
+    service: 'pashto',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/bbc_pashto_tv/tv_programmes/w13xttn4', // On Demand TV Brand
+    service: 'pashto',
+    runforEnv: ['local', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/w3ct2694', // On Demand Radio Episode
+    service: 'pashto',
+    runforEnv: 'local',
+    tests: canonicalTests,
+  },
+];
 
-const pageType = 'all';
+const canonicalTestSuites = Cypress.env('SMOKE')
+  ? smokeTestSuites
+  : nonSmokeTestSuites;
 
-Object.keys(config)
-  .filter(service => serviceFilter(service))
-  .forEach(service => {
-    const { variant } = config[service];
-
-    const paths = getPaths(service);
-    paths.forEach(path => {
-      describeForEuOnly(`${path} - Canonical Cookie Banner`, () => {
-        if (service !== 'news') {
-          runCanonicalTests({ service, variant, pageType, path });
-        }
-      });
-    });
-  });
+runTestsForPage({
+  pageType: 'all',
+  testSuites: [...canonicalTestSuites],
+});
```

Comment:
The cookie banner tests will only work in the EU i.e. in our CD / E2E pipelines - but they won't work on GitHub because the servers are based in the US. 

Previously the `describeForEuOnly` check was used to ensure that these tests only ran in EU
https://github.com/bbc/simorgh/blob/3b259452259ef0ca9d93a30300dbbee2f0a60924/cypress/support/helpers/describeForEuOnly.js#L5-L9

In order to replicate this behviour, we can add the same check here:

```suggestion
if (!Cypress.env('SKIP_EU')) {
  runTestsForPage({
    pageType: 'all',
    testSuites: [...canonicalTestSuites],
  });
}
```

Reply:
If you wanted to make these changes, we would use the Scheduled E2Es pipeline to test that these tests run & pass before we merge? Let me know & we can pair on it.